src/tests/mem/Mybuild

```diff
@@ -43,3 +43,10 @@ module mmap {
 	depends embox.kernel.task.resource.mmap_full
 	depends embox.compat.posix.sys.mman.mmap
 }
+
+module mprotect {
+	source "mprotect.c"
+
+	depends embox.arch.mmu
+	depends embox.compat.posix.sys.mman.mprotect
```

Comment:
Can you please remove dependencies from this test? We are trying to avoid test dependencies.

Reply:
Removed.